src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -178,6 +179,31 @@ void testMapPathologyRecordToBundleWithPerformingOrganization() {
         );
     }
 
+    @Test
+    void testMapPathologyRecordToBundleWithProcedureRequest() {
+        final var mockProcedureRequest = mock(ProcedureRequest.class);
+        pathologyRecordBuilder.testRequestSummary(mockProcedureRequest);
+
+        final var bundle = bundleMapper.mapToBundle(pathologyRecordBuilder.build());
+
+        final var procedureRequestBundleEntries = bundle.getEntry().stream()
+            .filter(entry -> entry.getResource() instanceof ProcedureRequest)
+            .collect(Collectors.toList());
+        final var procedureRequests = procedureRequestBundleEntries.stream()
+            .map(BundleEntryComponent::getResource)
+            .map(ProcedureRequest.class::cast)
+            .collect(Collectors.toList());
+
+        assertAll(
+            () -> verifyBundle(bundle),
+            () -> assertThat(procedureRequests).hasSize(1)
+                .contains(mockProcedureRequest),
```

Comment:
I was today years old when I realised the sequence `.hasSize(n).contains(n things...)` is the same as `.containsExactly(n things...)`
```suggestion
            () -> assertThat(procedureRequests)
                .containsExactly(mockProcedureRequest),
```